### DISCUSSION TITLE
What countries have birds on their flag?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#986] [Galaxy Trivia] [answered]
**Bonus:** List country name, flag image and bird species.

**Comment by Sriansh-raj** [comment]
Several countries have birds on their national flags. Here is a list of some of them:
Albania: Double-headed Eagle
Dominica: Imperial Parrot
Ecuador: Condor of the Andes
Guatemala: Quetzal
Kiribati: Frigate
Papua New Guinea: Bird of Paradise Raggiana
Uganda: Royal Grey Crane
Zimbabwe: Zimbabwe Bird

**Comment by peterbaccaro** [comment]
| Country Name | Bird Species | Flag Image |
| ------------- | ------------- | ------------- |
| Albania  | Eagle  | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/abd06e12-7bb6-4eef-8c90-e1b8e7c32497) |
| Austria  | Eagle  | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/984a0cac-eb6b-44b8-8c88-35f124a153ef) |
| Dominica |  Parrot | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/a5498403-5081-4589-8ba7-c48073fdb056) |
| Ecuador | Condor | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/762a531b-7a94-4b22-b6ef-51cfc8181bcb) |
| Egypt | Eagle | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/50e767c7-23ab-416b-8264-abc4f1a58797) |
| Fiji | Dove | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/db00657b-17c2-4ca9-809c-00e4ba68588f) |
| Guatemala | Quetzal | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/86e9ef7c-bc91-4270-915f-cf862dcf548d) |
| Kazakhstan | Eagle | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/53500dc4-0224-4027-80da-ae394663113b) |
| Kiribati | Frigate | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/f2118b87-1fa5-44a5-8369-6c86b09f5d48) |
| Mexico | Eagle | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/eba6ac05-0db1-4b46-a836-919523f69973) |
| Moldova | Eagle | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/b6d30cd2-9276-4e8e-99e3-0b02cc3edcbd) |
| Montenegro | Eagle | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/e66298e7-8c3b-4760-a035-058cb2c2f7f7) |
| Papua New Guinea | Bird of Paradise Raggiana | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/c4243cd1-6f8c-4723-bb66-2d112a1a363c) |
| Serbia | Eagle | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/a521879f-6048-4a18-aa43-64b531bfa4e8) |
| Uganda | Crane | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/3a583eef-c613-40d7-9151-c3319ba7d09b) |
| Zambia | Eagle | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/7e7e1470-c6bd-401e-8ae0-aa19cb73b9c0) |
| Zimbabwe | Zimbabwe Bird | ![image](https://github.com/ParanoidUser/yolo/assets/74666079/c21d70fe-139a-4bea-95f0-e973ff6d96e9) |

**Comment by thuongtruong1009** [comment, marked as answer]
Here are some countries that have birds featured on their national flags:

1. United States of America:
   Flag Image: ![USA Flag](https://upload.wikimedia.org/wikipedia/en/thumb/a/a4/Flag_of_the_United_States.svg/1920px-Flag_of_the_United_States.svg.png)
   Bird Species: Bald Eagle

2. Mexico:
   Flag Image: ![Mexico Flag](https://upload.wikimedia.org/wikipedia/commons/thumb/f/fc/Flag_of_Mexico.svg/1920px-Flag_of_Mexico.svg.png)
   Bird Species: Golden Eagle

3. Dominica:
   Flag Image: ![Dominica Flag](https://upload.wikimedia.org/wikipedia/commons/thumb/c/c4/Flag_of_Dominica.svg/1920px-Flag_of_Dominica.svg.png)
   Bird Species: Sisserou Parrot (Amazona imperialis)

4. Papua New Guinea:
   Flag Image: ![Papua New Guinea Flag](https://upload.wikimedia.org/wikipedia/commons/thumb/e/e3/Flag_of_Papua_New_Guinea.svg/1920px-Flag_of_Papua_New_Guinea.svg.png)
   Bird Species: Raggiana Bird-of-Paradise (Paradisaea raggiana)

5. Kiribati:
   Flag Image: ![Kiribati Flag](https://upload.wikimedia.org/wikipedia/commons/thumb/d/d3/Flag_of_Kiribati.svg/1920px-Flag_of_Kiribati.svg.png)
   Bird Species: Frigatebird (Fregata spp.)

6. Zimbabwe:
   Flag Image: ![Zimbabwe Flag](https://upload.wikimedia.org/wikipedia/commons/thumb/6/6a/Flag_of_Zimbabwe.svg/1920px-Flag_of_Zimbabwe.svg.png)
   Bird Species: African Fish Eagle (Haliaeetus vocifer)

Please note that the bird species mentioned here are representative of the countries' flags and may not be the only bird species associated with these countries.
